### ISSUE TITLE
Percentage labels example causes Typescript errors

### ISSUE THREAD

**Issue by majelbstoat** [toomuchdesign/react-minimal-pie-chart#154]
### Do you want to request a _feature_ or report a _bug_?

report a _bug_
...

### What is the current behaviour?

set props like the Percentage labels story:

```tsx
<PieChart
      data={dataMock}
      label={({ data, dataIndex }) =>
        Math.round(data[dataIndex].percentage) + '%'
      }
      labelStyle={defaultLabelStyle}
    />
```

Typescript complains with:

```
Type '({ data, dataIndex }: Props) => string' is not assignable to type 'boolean | (false & LabelPropAsReactElement) | (false & LabelPropAsFunction) | (true & LabelPropAsReactElement) | (true & LabelPropAsFunction) | undefined'.
  Type '({ data, dataIndex }: Props) => string' is not assignable to type 'false & LabelPropAsFunction'.
    Type '({ data, dataIndex }: Props) => string' is not assignable to type 'false'.ts(2322)
```
...

### What is the expected behaviour?

The example works as expected.
...

### Steps to Reproduce the Problem

1. use `label={({ data, dataIndex }) => Math.round(data[dataIndex].percentage) + '%'}`
2. Observe the error in VSCode
### Specifications

- Version: 7.3.0
- Platform: Typescript 3.8.3

**Comment by majelbstoat** [comment]
This appears to be because the generated types in Chart.d.ts look like:

```tsx
export declare type Props = typeof ReactMinimalPieChart.defaultProps & {
 // ...
 label?: boolean;
 // ...
}
```

If I change them to:

```tsx
export declare type Props = typeof ReactMinimalPieChart.defaultProps & {
 // ...
 label?: LabelProp;
 // ...
}
```

Typescript doesn't complain anymore.

**Comment by toomuchdesign** [comment]
Hopefully fixed by v7.3.1.
